src/In.ProjectEKA.FHIRHip/DataFlow/Model/NetworkDataRequest.cs

```diff
@@ -0,0 +1,21 @@
+namespace In.ProjectEKA.DefaultHip.DataFlow.Model
+{
+    using System.Collections.Generic;
+    using HipLibrary.Patient.Model;
+
+    public class NetworkDataRequest
+    {
+        public string PatientReference { get; set; }
+        public IEnumerable<string> CareContexts { get; set; }
+        public DateRange DataRange { get; set; }
```

Comment:
From and To date would be strings in this case. Shouldn't it be DateTime?

Reply:
It is similar to as of in data request. We are fetching it from data request. So kept it same